core/modules/wdb/QueryRunner.h

```diff
@@ -126,8 +138,14 @@ class QueryRunner : public wbase::TaskQueryRunner, public std::enable_shared_fro
 
     util::MultiError _multiError; // Error log
 
+#ifdef QSERV_USE_PROTO_ARENA
+    std::unique_ptr<google::protobuf::Arena> _arena;
+    proto::ProtoHeader* _protoHeader = nullptr;
+    proto::Result* _result = nullptr;
+#else
     std::shared_ptr<proto::ProtoHeader> _protoHeader;
     std::shared_ptr<proto::Result> _result;
+#endif
```

Comment:
I think it's safe to just get rid of the code that doesn't use the arena and the QSERV_USE_PROTO_ARENA define.

Reply:
Fixed.